src/SourceLocator/Type/Composer/Factory/MakeLocatorForComposerJsonAndInstalledJson.php

```diff
@@ -169,9 +174,9 @@ private function packageToFilePaths(array $package): array
     /**
      * @param array{name: string} $package
      */
-    private function packagePrefixPath(string $trimmedInstallationPath, array $package): string
+    private function packagePrefixPath(string $trimmedInstallationPath, array $package, string $vendorDir = 'vendor'): string
```

Comment:
`$vendorDir` no longer needs a default, since it is always declared and passed in explicitly